site/app/templates/grading/electronic/Details.twig

```diff
@@ -64,14 +64,22 @@
             var button = document.getElementById('inquiryButton');
             status = Cookies.get('inquiry_status')
             if (status === 'on') {
-                grade_inquiry_only();
-                button.innerHTML = "Grade Inquiry Only: On";
-            } else {
-                reverse_inquiry_only();
-                button.innerHTML = "Grade Inquiry Only: Off";
+                $('[data-testid=grade-button]').each(function() {
+                    if (typeof $(this).attr('data-grade-inquiry') === 'undefined') {
+                        $(this).closest('[data-testid="grade-table"]').hide();  // hide gradeable items without active inquiries
+                    }
+                });
```

Comment:
Is there a better way to query these elements?

Reply:
I don't think there is a better way to query those. 
![image](https://github.com/Submitty/Submitty/assets/114595115/b081c56e-2883-407e-8f61-c4319e8029c1)
since I'm trying to show/hide the entire row. I think I have to query '[data-testid="grade-table"]'